docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -11,61 +14,289 @@ Users deploying their Universal Profiles using the guides that utilize **[lsp-fa
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
+import {
+  ERC725YDataKeys,
+  PERMISSIONS,
+} from '@lukso/lsp-smart-contracts/constants.js';
+import Web3 from 'web3';
 
+// constants
 const web3 = new Web3('https://rpc.l16.lukso.network');
+const URD_DATA_KEY = ERC725YDataKeys.LSP1.LSP1UniversalReceiverDelegate;
+const universalProfileAddress = '0x...';
+
+// setup your EOA
+const privateKey = '0x...';
+const EOA = web3.eth.accounts.wallet.add(privateKey);
+```
 
-const PRIVATE_KEY = '0x...'; // your EOA private key
-const myEOA = web3.eth.accounts.wallet.add(PRIVATE_KEY);
+  </TabItem>
 
-// create an instance
-let myURD = new web3.eth.Contract(LSP1UniversalReceiverDelegateUP.abi);
+  <TabItem value="ethersjs" label="ethers.js">
+
+```typescript title="Imports, Constants & EOA"
+import LSP1UniversalReceiverDelegateUP from '@lukso/lsp-smart-contracts/artifacts/LSP1UniversalReceiverDelegateUP.json';
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
+import {
+  ERC725YDataKeys,
+  PERMISSIONS,
+} from '@lukso/lsp-smart-contracts/constants.js';
+import { ethers } from 'ethers';
+
+// constants
+const provider = new ethers.JsonRpcProvider('https://rpc.l16.lukso.network');
+const URD_DATA_KEY = ERC725YDataKeys.LSP1.LSP1UniversalReceiverDelegate;
+const universalProfileAddress = '0x...';
+
+// setup your EOA
+const privateKey = '0x...'; // your EOA private key (controller address)
+const EOA = new ethers.Wallet(privateKey).connect(provider);
+```
 
-// deploy the URD contract
-await myURD
+  </TabItem>
+
+</Tabs>
+
+## Step 2 - Deploy the default Universal Receiver Delegate contract
+
+:::info
+
+The **Universal Profile** and the **Vault** don't use the same implementation of the Universal Receiver Delegate.
+:::
+
+### Step 2.1 - Create a contract instance
+
+At this step we will create an instance of the Universal profile URD that we will further be used to deploy one.
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```typescript title="Contract instance of the Universal Profile URD"
+// create an instance of the LSP1UniversalReceiverDelegateUP
+let universalProfileURD = new web3.eth.Contract(
+  LSP1UniversalReceiverDelegateUP.abi,
+);
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```typescript title="Contract instance of the Universal Profile URD"
+// create a LSP1UniversalReceiverDelegateUP Contract Factory
+let universalProfileURDFactory = new ethers.ContractFactory(
+  LSP1UniversalReceiverDelegateUP.abi,
+  LSP1UniversalReceiverDelegateUP.bytecode,
+);
+```
+
+  </TabItem>
+
+</Tabs>
+
+### Step 2.2 - Send the contract deployment transaction
+
+Send the deployment transaction and in a few seconds you will get a new deployed Unviersal Profile URD.
```

Comment:
Not sure if we should write _"in a few seconds"_ as it might depend on the state of the network.

```suggestion
Send the deployment transaction to get a newly deployed URD.
```